tool/planet/cfg.go

```diff
@@ -154,12 +156,39 @@ func (cfg *Config) checkAndSetDefaults() (err error) {
 	if err != nil {
 		return trace.Wrap(err)
 	}
+
+	if err := verifyPodSubnetSize(cfg.PodSubnetSize, cfg.PodCIDR); err != nil {
+		return trace.Wrap(err, "failed to verify pod subnet size")
+	}
+
 	if cfg.VxlanPort <= 0 {
 		cfg.VxlanPort = DefaultVxlanPort
 	}
 	return nil
 }
 
+// verifyPodSubnetSize verifies that the subnet size is not too small and verifies
+// that the subnet size is not larger than the network CIDR range.
+func verifyPodSubnetSize(subnetSize int, cidr kv.CIDR) error {
+	// Verify requested subnet size is not too small.
+	if subnetSize > 30 {
```

Comment:
/30 will give 4 pods per node max? I don't think even our bare base image install will fit. @knisbet What do you think would be a reasonable minimum? We could probably use whatever customer S requested as a reference, like I would probably consider /27 or at most (least?) /28 as the minimum reasonable size.

Reply:
/28 appears to be the minimum accepted by flannel: https://github.com/gravitational/flannel/blob/master/subnet/config.go#L70-L74

/28 does seem too small for a normal gravity install, but would probably be OK when disabling most the builtin apps. So I would probably start with a /28.